gulpfile.js/lib/pathToUrl.js

```diff
@@ -0,0 +1,6 @@
+var path = require('path')
+
+module.exports = function pathToUrl() {
+  // Normalizes Windows file paths to valid url paths
+  return path.normalize(path.join.apply(this, arguments).replace(/\\/g, '/'))
```

Comment:
OOOOooohhhh. right. Normalize uses the system slashes (which undos the string replace). I was thinking it just combined double slashes.